src/PaymentServer/Processors/Stripe.hs

```diff
@@ -11,22 +11,39 @@ module PaymentServer.Processors.Stripe
 import Control.Monad.IO.Class
   ( liftIO
   )
+import Control.Monad
+  ( mzero
+  )
+import Data.ByteString
+  ( ByteString
+  )
+import Data.Text
+  ( Text
+  , unpack
+  )
 import Text.Printf
   ( printf
   )
 import Data.Aeson
   ( ToJSON(toJSON)
+  , FromJSON(parseJSON)
+  , Value(Object)
   , object
+  , (.:)
   )
 import Servant
   ( Server
   , Handler
+  , err400
+  , ServerError(errBody)
+  , throwError
   )
 import Servant.API
   ( ReqBody
   , JSON
   , Post
   , (:>)
+  , (:<|>)(..)
```

Comment:
Elsewhere I did:

```
 , (:<|>)((:<|>)) 
```
which is quote a mouthful but I still tend to like to avoid `(..)`.  In this case it probably doesn't matter since the type and the constructor have the same name so it's easy to search for.  But, I dunno, maybe `:<|>` has another type constructor I don't know about that this is making available and that will lead to confusion later on.